### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by nervo**
By working on a pull request, i've noticed few typos across the repo :)

### CODE REVIEW

**Comment by muesli** [comment]
Thank you!

File: position.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ func Place(width, height int, hPos, vPos Position, str string, opts ...Whitespac
 
 // PlaceHorizontal places a string or text block horizontally in an unstyled
 // block of a given width. If the given width is shorter than the max width of
-// the string (measured by it's longest line) this will be a noöp.
+// the string (measured by its longest line) this will be a noop.
```
Comment (meowgorithm) [inline position.go:42]: Farewell my diaeresis

Comment (nervo) [reply position.go:42]: @meowgorithm don't tell me it was intentional :)

Comment (meowgorithm) [reply position.go:42]: @nervo Alas, as a native English speaker, this was my chance to flaunt the only diacritic we (barely) have. It was naïve of me to think spell checkers would coöperate accordingly.

:D

Comment (nervo) [reply position.go:42]: Now i feel both sad and ashamed...
Here is a lil' duck for you, may its cuteness fill your heart

**Pull request merged**